Update enroll policy doc for new authenticators format (#2585)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

   }
```

#### Policy Authenticators Settings example

<ApiLifecycle access="ie" /><br>
<ApiLifecycle access="Limited GA" /><br>

> **Note:** Policy Settings are included only for those Authenticators that are enabled.

```json
   "settings": {
     "type": "AUTHENTICATORS",
     "authenticators": [
       {
         "key": "security_question",
         "enroll": {
           "self": "OPTIONAL"
         }
       },
       {
         "key": "okta_phone",
         "enroll": {
           "self": "OPTIONAL"
         }
       }
     ]
   }
```

### Policy Settings data

> > **Note:** Default type is `FACTORS` when not specified.

Reviewer: Instead of this, just add a `Default` column to the table like we do in other parameter tables

![image](https://user-images.githubusercontent.com/33136962/134256016-b3a23e9a-fcc3-4d2b-9b15-c313465036f3.png)

> > **Note:** Unlike the `factors` parameter which only controls MFA methods, `authenticators` will have all authenticators available; both authentication and recovery.

Reviewer 2: Maybe this note should go after the parameters table? Referring to `factors` and `authenticators` after the table will give users some context.

Also, do you want to change the note to:

```suggestion
> **Note:** The `authenticators` parameter allows you to configure all available Authenticators, including authentication, enrollment, and recovery. In contrast, the `factors` parameter only allows you to configure multifactor authentication.
```
This is just a suggestion to emphasize the `authenticators` parameter features over the `factors` one. Thanks!

| Parameter                                                               | Description                                                                  | Data Type                      | Required |
| ---                                                                     | ---                                                                          | ---                            | ---      |
| type <ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" />  | Type of policy configuration object                                                | `FACTORS`, or `AUTHENTICATORS` | No       |

> | factors                                                                 | [Policy Factors Configuration object](#policy-factors-configuration-object)  | Array                          | No       |

Reviewer: Shouldn't `Data Type` for factors by Object, not Array?

> | authenticators                                                          | Array of [Policy MFA Authenticator object](#policy-mfa-authenticator-object) | Array                          | No       |

Reviewer: Can we add the `<ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" />` tags here too

         }
       }
     ]
   }
```

### Policy Settings data

| Parameter                                                                        | Description                           | Data Type                                                                    | Required | Default   |
| ---                                                                              | ---                                   | ---                                                                          | ---      | ---       |
| authenticators <ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" /> | List of Authenticator policy settings | Array of [Policy Authenticator object](#policy-authenticator-object) | No       |           |
| factors                                                                          | Factor policy settings                | [Policy Factors Configuration object](#policy-factors-configuration-object)  | No       |           |
| type <ApiLifecycle access="ie" /> <ApiLifecycle access="Limited GA" />           | Type of policy configuration object   | `FACTORS` or `AUTHENTICATORS`                                                | No       | `FACTORS` |

> > **Note:** The `authenticators` parameter allows you to configure all available Authenticators, including authentication, and recovery. In contrast, the `factors` parameter only allows you to configure multifactor authentication.

Reviewer 2: ```suggestion
> **Note:** The `authenticators` parameter allows you to configure all available Authenticators, including authentication and recovery. In contrast, the `factors` parameter only allows you to configure multifactor authentication.
```

#### Policy MFA Authenticator object

<ApiLifecycle access="ie" /><br>
<ApiLifecycle access="Limited GA" /><br>

| Parameter | Description                                               | Data Type                                                            | Required |
| ---       | ---                                                       | ---                                                                  | ---      |
| key       | A human-readable string that identifies the Authenticator | String                                                               | Yes      |
| enroll    | Enrollment requirements for the Authenticator             | [Policy Authenticator Enroll object](#policy-authenticator-enroll-object)   | Yes      |

#### Policy Authenticator Enroll object

<ApiLifecycle access="ie" /><br>
<ApiLifecycle access="Limited GA" /><br>

| Parameter | Description                                | Data Type                               | Required | Default       |
| ---       | ---                                        | ---                                     | ---      | ---           |

> | self      | Requirements for User-initiated enrollment | `NOT_ALLOWED`, `OPTIONAL` or `REQUIRED` | No       | `NOT_ALLOWED` |

Reviewer 2: ```suggestion
| self      | Requirements for the user-initiated enrollment | `NOT_ALLOWED`, `OPTIONAL`, or `REQUIRED` | No       | `NOT_ALLOWED` |
```
